internal/js/modules/k6/browser/common/http.go

```diff
@@ -161,13 +161,13 @@ func NewRequest(ctx context.Context, logger *log.Logger, rp NewRequestParams) (*
 		wallTime:            ev.WallTime.Time(),
 		offset:              ev.WallTime.Time().Sub(ev.Timestamp.Time()),
 		documentID:          documentID.String(),
-		headers:             make(map[string][]string),
+		headers:             make(map[string][]string, len(ev.Request.Headers)),
 		ctx:                 ctx,
 		vu:                  k6ext.GetVU(ctx),
 	}
 	for n, v := range ev.Request.Headers {
 		if s, ok := v.(string); ok {
-			r.headers[n] = append(r.headers[n], s)
+			r.headers[n] = []string{s}
```

Comment:
Is this correct? Aren't we going to lose the previously appended values in the slice if it already existed?

Reply:
After reading it again, i think this is fine.